test/test_subscription.py

```diff
@@ -321,3 +333,61 @@ def test_continuous_subscription(self):
                 self.retrieve(8889, fname, fileVersion=2, targetFile=tmp_path())
         finally:
             subscription_listener.close()
+
+    if ssl is not None:
+        @unittest.skipIf('NGAS_TESTDB' in os.environ, "Need two cluster for this test to run")
+        def test_https_subscription(self):
+
+            ca = CAHandler()
+            cert_file = ca.create_server_certificate(
+                [u"localhost", six.u(getHostName())],
+                [ipaddress.IPv4Address(u"127.0.0.1")],
+            ).combined_filename
+            # We configure the second server to send notifications via socket
+            # to the listener we start later
+            cfg = self.env_aware_cfg()
+            cfg.storeVal(
+                'NgamsCfg.ArchiveHandling[1].EventHandlerPlugIn[1].Name',
+                'test.ngamsTestLib.SenderHandler'
+            )
+
+            tmpCfgFile = genTmpFilename(suffix='xml')
+            cfg.save(tmpCfgFile, 0)
+            self.prepExtSrv(port=8778, cert_file=cert_file)
+            self.prepExtSrv(port=8779, cert_file=cert_file, cfgFile=tmpCfgFile)
```

Comment:
Instead of creating a configuration file, saving it, and passing it via `cfgFile` you can pass down a list of 2-element tuples directly to prepExtSrv via `cfgProps` and it will do that for you. Shouldn't change the behavior in any way, just make it easier to read the test code.